### PR TITLE
Add documentation about poetry environment and installation

### PR DESCRIPTION
**Pull request by bkorycki**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,13 +43,14 @@ install ModelBench as a CLI tool or library to use in your own projects.
 ```shell
 pipx install poetry
 ```
+Please be sure to install Poetry in a dedicated python virtual environment separate from the environment used for this project.
```
Comment (wpietri) [inline README.md:46]: This is a surprise to me. Doesn't pipx create its own environment?

If it is something you want people to do, you'll have to give more detail.

File: README.md [hunk 2]
```diff
@@ -43,13 +43,14 @@ install ModelBench as a CLI tool or library to use in your own projects.
...
 
 2. Clone this repository.
 ```shell
 git clone https://github.com/mlcommons/modelbench.git
 ```
 
-3. Install ModelBench and dependencies.
+3. Install ModelBench and dependencies. Make sure the dedicated python environment you used to install Poetry is not activated when running poetry commands.
```
Comment (wpietri) [inline README.md:53]: What's written here seems to me not to be the case. I haven't had problems running poetry from the project's environment, or from the default one. If there is something you want people to do, you'll at least have to be more explicit about it, and possibly mention the error message they'd be seeing if they don't do what you want.

**Review by wpietri — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (dhosterman) [reply README.md:46]: Pipx does create its own virtual environment(s) for everything you install in it.

File: README.md (hunk 2 above)
Comment (bkorycki) [reply README.md:53]: It's not the project's environment that's the problem, it's the environment that was created to install poetry. This is based on the installation section of the poetry docs https://python-poetry.org/docs/

**Review by bollacker — APPROVED** [review]
LGTM

**Pull request closed without merging**